lib/media/streaming_engine.js

```diff
@@ -1644,8 +1644,11 @@ shaka.media.StreamingEngine = class {
         'bufferBehind=' + bufferBehind,
         'overflow=' + overflow);
 
-    await this.playerInterface_.mediaSourceEngine.remove(mediaState.type,
-        startTime, startTime + overflow);
+    // See: https://github.com/google/shaka-player/issues/2982
+    if (overflow > 0.01) {
```

Comment:
I think it would be better to do this check on 1630 so the logs are correct.

Reply:
Done